handler/pullRequestHandler.go

```diff
@@ -211,6 +260,22 @@ func fetchPullRequestCommits(req types.PullRequestOuter, client *github.Client)
 	return commits, nil
 }
 
+func fetchPullRequestFiles(req types.PullRequestOuter, client *github.Client) ([]*github.CommitFile, error) {
+	ctx := context.Background()
+	listOpts := &github.ListOptions{
+		Page: 0,
+	}
+	commitFiles, resp, err := client.PullRequests.ListFiles(ctx, req.Repository.Owner.Login, req.Repository.Name, req.PullRequest.Number, listOpts)
```

Comment:
Are we sure that isn't part of the JSON already?

Reply:
Yes, we found that out the hard way. The check wasn't running. After looking at the request from GH, we saw that the `files` were a separate call

Reply:
I was wondering if we could avoid the extra API call to check files.

Reply:
Do any files get downloaded, or is it just a list?

`fetchPullRequestFiles` makes me feel like the disk is written too. If it's not written to, can we just go with `listPullRequestFiles` or whatever you prefer?